### DISCUSSION TITLE
please make a option to make it so when you walk in a sideways or upside down portal it changes your gravity

### DISCUSSION THREAD

**Discussion by squendan998** [iPortalTeam/ImmersivePortalsMod#1011] [Ideas]
please make a option to make it so when you walk in a sideways or upside down portal it changes your gravity

**Comment by qouteall** [comment]
This option exists since 1.18. You can see a command stick that enables gravity chagne to a portal

**Reply by squendan998** [reply]
i know but im saying i want you to make it so when you walk in the upside down portal you dont do a front flip

**Reply by squendan998** [reply]
i want to instead be able to actualy flip my gravity when i walk in it

**Reply by squendan998** [reply]
but is it in the 1.17 version

**Reply by squendan998** [reply]
thats the one im always on

**Reply by squendan998** [reply]
i just switched to 1.18.1

**Reply by squendan998** [reply]
OMG YOU DIDN'T LIE

**Reply by squendan998** [reply]
THANK YOU SO MUCH
